scripts/mea/et/internal_witness.py

```diff
@@ -274,6 +274,29 @@ def process_verifier_notes(self):
                 env = self._env_models_json[file_id][start_line]
                 comment = env["comment"]
                 relevant = env.get("relevant", False)
+                name = env.get("name", None)
+                action_type = env.get("type", None)
+                self._logger.debug(f"type: {action_type} for name {name}")
+                action_type_data = {}
+                if name is not None:
+                    action_type_data[name] = []
+                if action_type == "ACTION_BEGIN":
+                    action_begin_dict = {}
+                    action_begin_dict['env'] = comment
+                    action_begin_dict['file'] = file_id
+                    action_begin_dict['source'] = edge["source"]
```

Comment:
Are you sure this tag is presented there? You need to check first, if it is present and fail correspondingly otherwise